lib/Serverless.js

```diff
@@ -87,7 +87,8 @@ class Serverless {
       this._shouldReportMissingServiceDeprecation = true;
     }
     const commands = ensureArray(configObject.commands, { isOptional: true });
-    const options = ensurePlainObject(configObject.options, { isOptional: true });
+    let options = ensurePlainObject(configObject.options, { isOptional: true });
+    if (options) options = { ...options };
```

Comment:
Let's maybe add a comment on why we're doing that (technically it's a temporary hack until we have https://github.com/serverless/serverless/issues/2582 addressed)